website/assets/resources/install-fleetctl.sh

```diff
@@ -27,8 +27,14 @@ version_gt() {
 # Determine operating system (Linux or MacOS)
 OS="$(uname -s)"
 
+# Determine architecture (x86_64 or arm64)
+ARCH="$(uname -m)"
+# Standardize x86_64 to amd64
+if [ $ARCH != "arm64" ]; then ARCH="amd64"; fi
```

Comment:
Probably safer to also check for the four possible values for `arm64`: `aarch64`, `aarch64_be`, `armv8b` and `armv8l` (see [here](https://stackoverflow.com/questions/45125516/possible-values-for-uname-m))?

PS: My Raspberry Pi returns `aarch64` for `uname -m`.

Reply:
Done, now you'll be able to run the Fleet website with online preview from your Raspberry Pi! 🥳

Reply:
lol, sorry, probably not a big deal, but safe to fix in case we replicate/copy this script.